docs/enterprise/index.md

```diff
@@ -0,0 +1,103 @@
+---
```

Comment:
Any idea why these pages are showing up as new additions?

Reply:
Sorry, this came from my end. 
Same problem [as above](https://github.com/ethereum/ethereum-org-website/pull/246#discussion_r326344209). I hadn't rebased yet and I didn't check to see what `git add .` was adding.

I belive my last rebase+push fixed this.